### PR TITLE
Removed financial-oriented series default accessors

### PR DESCRIPTION
**Pull request by ColinEberhardt**
d3fc is not just for financial charting! For that reason I am proposing changing the accessor defaults for non financial-specific series types.

### CODE REVIEW

**Comment by ColinEberhardt** [comment]
See #572

File: src/series/boxPlot.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ export default function() {
...
         yScale = d3.scale.linear(),
-        upperQuartile = function(d, i) { return (d.close + d.high) / 2; },
-        lowerQuartile = function(d, i) { return (d.close + d.low) / 2; },
+        upperQuartile = function(d, i) { return d.upperQuartile; },
+        lowerQuartile = function(d, i) { return d.lowerQuartile; },
         high = function(d, i) { return d.high; },
         low = function(d, i) { return d.low; },
-        value = function(d, i) { return d.date; },
-        median = function(d, i) { return d.close; },
+        value = function(d, i) { return d.x; },
```
Comment (chrisprice) [inline src/series/boxPlot.js]: Should ```x`` be the default or should it be ```value```?

File: src/series/errorBar.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export default function() {
         yScale = d3.scale.linear(),
         high = function(d, i) { return d.high; },
         low = function(d, i) { return d.low; },
-        value = function(d, i) { return d.date; },
+        value = function(d, i) { return d.x; },
```
Comment (chrisprice) [inline src/series/errorBar.js]: Same as above.

**Comment by ColinEberhardt** [comment]
@chrisprice good points - updated

**Comment by chrisprice** [comment]
:+1:

**Pull request merged**